gpytorch/models/gp.py

```diff
@@ -1,7 +1,14 @@
 #!/usr/bin/env python3
 
+from torch import Tensor
+
 from ..module import Module
 
 
 class GP(Module):
-    pass
+    def apply_input_transforms(self, X: Tensor, is_training_input: bool) -> Tensor:
```

Comment:
Is there a reason to not name this arg `is_training` rather than `is_training_input`?

Reply:
Just being verbose and differentiating the model being in training and the inputs being the `train_inputs` or otherwise being treated as such.